### PR TITLE
Fixed Login pop up

### PR DESCRIPTION
**Pull request by TinelValentin**

### CODE REVIEW

**Comment by TinelValentin** [comment]
Also fixed a bug where the styles where not always used in sign up view

**Review by BrassoiCatalin — APPROVED** [review]

File: SevenSlots/SevenSlots/SevenSlots/View/LoginErrorPopup.xaml [hunk 1]
```diff
@@ -2,16 +2,24 @@
...
     <ContentPage.Content>
-        <StackLayout BackgroundColor="Red" 
+        <StackLayout BackgroundColor="White" 
                      HorizontalOptions="Center"
                      VerticalOptions="Center" 
-                     Margin="20" 
-                     Spacing="0">
-            <Label Text="Wrong credentials!"
-                   HorizontalOptions="Center" 
+                     Margin="20"
```
Comment (Sorin37) [inline SevenSlots/SevenSlots/SevenSlots/View/LoginErrorPopup.xaml:11]: align the attributes accordingly

File: SevenSlots/SevenSlots/SevenSlots/View/SignUpView.xaml [hunk 2]
```diff
@@ -60,10 +60,11 @@
                             <Frame Style="{StaticResource InputFrame}"
                                     x:Name="usernameFrame"/>
                             <controls:CustomInputBox
+                                Style="{StaticResource CustomInputStyle}"
                                 x:Name="usernameInput"
                                 Placeholder="Username"
                                 Text="{Binding User.Username,Mode=TwoWay}"
-                                Style="{StaticResource CustomInputStyle}" >
+                                >
```
Comment (Sorin37) [inline SevenSlots/SevenSlots/SevenSlots/View/SignUpView.xaml]: inconsistent tag ending

File: SevenSlots/SevenSlots/SevenSlots/View/SignUpView.xaml [hunk 3]
```diff
@@ -60,10 +60,11 @@
...
                                 <Entry.Behaviors>
                                     <xct:MultiValidationBehavior 
                                          InvalidStyle="{StaticResource InvalidStyle}">
```
Comment (Sorin37) [inline SevenSlots/SevenSlots/SevenSlots/View/SignUpView.xaml:69]: this can go on the same line with the tag

**Review by Sorin37 — CHANGES_REQUESTED** [review]

File: SevenSlots/SevenSlots/SevenSlots/View/LoginErrorPopup.xaml (hunk 1 above)
Comment (TinelValentin) [reply SevenSlots/SevenSlots/SevenSlots/View/LoginErrorPopup.xaml:11]: But it is alligned

**Review by Sorin37 — APPROVED** [review]

**Pull request merged**